packages/cli/config.d.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 The Backstage Authors
```

Comment:
This config currently lives in https://github.com/backstage/backstage/blob/eef0522644ed9d50fcc7fd69e7a712c2719b47ab/packages/cli/package.json#L182 so I think we should either add it to the `package.json` or move the existing one into here and remove the one in the `package.json`. Let's keep a single source of truth for this.